apps/yggdrasil/lib/yggdrasil/room.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Yggdrasil.Room do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Yggdrasil.Room
+
+  schema "rooms" do
+    field :title, :string
+    field :description, :string
+    field :is_starting, :boolean, default: false
+    timestamps
+  end
+
+  def create_changeset(params) do
+    %Room{ }
```

Comment:
I have questioned whether or not we should do this over the convention that I see where `*changeset` functions take in the model.

The usual way this is defined is like this:
```Elixir
def create_changeset(model, params) do
  model
  |> ...
end
```
that idiom is every I have seen, the default changeset for example is defined with the same parameters. If I had to guess it's for pipelining.